packages/ods-react/src/components/modal/src/components/modal-header/modalHeader.module.scss

```diff
@@ -19,7 +19,7 @@
     &--primary {
       @include modal.ods-modal-color('primary');
 
-      button {
+      .modal__close {
```

Comment:
`&__close`
(actually here you will have to do like in message, with a var reference on top `#{$message}__close`)

Reply:
this is why you descend from parent name